src/components/ChatLog.js

```diff
@@ -0,0 +1,41 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry.js';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ChatLog = ({ entries, likesCountCallBack }) => {
+  const chatComponents = entries.map((chat) => {
+    return (
+      <ChatEntry
+        key={chat.timeStamp}
+        id={chat.id}
+        sender={chat.sender}
+        body={chat.body}
+        timeStamp={chat.timeStamp}
+        liked={chat.liked}
+        likesCountCallBack={likesCountCallBack}
+      />
+    );
+  });
+
+  return (
+    <div>
+      <ul> {chatComponents} </ul>
+    </div>
```

Comment:
Style: I can't remember if the styles need it, but you don't need a surrounding `div` since you already have a surrounding `ul`.